packages/search/addon/components/navi-search-bar.hbs

```diff
@@ -1 +1,15 @@
-{{yield}}
+{{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<div class="navi-search" ...attributes>
+  <BasicDropdown @matchTriggerWidth={{true}} @verticalPosition="below" as |dd|>
+  <div class="navi-search-bar">
```

Comment:
dd.trigger?

Reply:
I don't use a trigger because I don't want the dropdown to open on click, only when a search is initiated (https://ember-basic-dropdown.com/cookbook/no-trigger)